LuaUI/Configs/integral_menu_config.lua

```diff
@@ -421,12 +422,20 @@ for i = 1, 5 do
 	}
 end
 
+local factoryButtonLayoutOverride = {}
+	factoryButtonLayoutOverride[1] = {
+		[3] = {
+			buttonLayoutConfig = buttonLayoutConfig.command,
+			isStructure = false,
+		}
+	}
```

Comment:
Could initialize the [1] right away instead of a separate assignment:
```suggestion
local factoryButtonLayoutOverride = {
	{
		[3] = {
			buttonLayoutConfig = buttonLayoutConfig.command,
			isStructure = false,
		}
	}
}
```